tests/shared/utils/test_io.py

```diff
@@ -620,3 +623,21 @@ def test_deep_container_fingerprint_can_use_instance_fingerprint():
     m1 = np.asarray([[0.5, 3.1, 3.0], [1.1, 1.2, 1.3], [4.7, 0.3, 2.7]])
     f = Features(m1, "sentence", "text", "CountVectorsFeaturizer")
     assert rasa.shared.utils.io.deep_container_fingerprint(f) == f.fingerprint()
+
+
+# Can't manipulate the STDOUT file descriptor in pytest (it's a io.StringIO instance not a file)
```

Comment:
Also this linting error causes the code quality check to fail. I'll suggest running `make lint` locally on your PRs and fixing such issues before merge.

Reply:
Thanks for pointing it out didn't have ruff installed...